src/test/ui/parser/let-binop.stderr

```diff
@@ -0,0 +1,39 @@
+error[E0067]: can't reassign to a uninitialized variable
+  --> $DIR/let-binop.rs:2:15
+   |
+LL |     let a: i8 *= 1;
+   |               ^^ help: replace with `=` to initialize the variable
+
+error[E0067]: can't reassign to a uninitialized variable
+  --> $DIR/let-binop.rs:4:11
+   |
+LL |     let b += 1;
+   |           ^^
+   |
+help: replace with `=` to initialize the variable
+   |
+LL |     let b = 1;
+   |           ^
+help: remove to reassign to a previously initialized variable
+   |
+LL |      b += 1;
+   |    --
```

Comment:
The suggestion span is slightly incorrect, as it should remove up to the start of `b`.